cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
```

Comment:
Can we maybe pull the encoder/decoders (these plus the older ones) into a separate object to give it a more clear namespace than the random grab bag of fun a package object provides?